### PR TITLE
[TASK] Create AUD funding page (RT-3059)

### PR DESCRIPTION
**Pull request by Mesrop**

### CODE REVIEW

**Comment by mrajvanshy** [comment]
LGTM

**Comment by yongsoo** [comment]
Please rebase @Mesrop

**Comment by jublo** [comment]
@Mesrop Could you be so kind as to squash those 3 commits? I think that would produce a cleaner history for us.

File: src/jade/tabs/aud.jade [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          .title(l10n) Withdrawal Options
+          .description(l10n)
+            | Coinex accepts withdrawals by the following methods:
+            ul
+              li
+                p(l10n) Domestic or international bank transfer
+              li
+                p(l10n) Ripple transaction
+              li
+                p(l10n) Stellar transaction
```
Comment (jublo) [inline src/jade/tabs/aud.jade]: I'd like to see a trailing newline here.

Comment (jublo) [inline src/jade/tabs/aud.jade]: @annatonger Do we want to make mention of Stellar anywhere in Ripple Trade? :question:

**Comment by Mesrop** [comment]
@jublo squash done, Stellar discussed here
https://github.com/ripple/ripple-client/pull/2106
I think its ok.

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**